docs/installing.rst

```diff
@@ -148,8 +148,22 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+---------------------------------------------------
+3. Install dependencies for vagrant-service-manager
```

Comment:
@bexelbie I mean keeping the information at one place will help maintaining it better.  We should document it here and reference it in VSM because the user facing docs should be here as ADB acts as landing page for the ecosystem.